source/_integrations/lock.mqtt.markdown

```diff
@@ -20,6 +20,8 @@ Optimistic mode can be forced, even if state topic is available. Try to enable i
 
 It's mandatory for locks to support `lock` and `unlock`. A lock may optionally support `open`, (e.g. to open the bolt in addition to the latch), in this case, `payload_open` is required in the configuration. If the lock is in optimistic mode, it will change states to `unlocked` when handling the `open` command.
 
+`unlocking`, `locking` and `jammed` are also supported.
+
```

Comment:
```suggestion

```suggestion
A MQTT lock can also report the intermediate states `unlocking`, `locking` and a state when the lock is `jammed`.

```